src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -0,0 +1,687 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:android="http://uno.ui/android"
+                    xmlns:ios="http://uno.ui/ios"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:not_mobile="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:todo="todo"
+                    xmlns:toolkit="using:Uno.UI.Toolkit"
+                    mc:Ignorable="ios android todo">
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary  x:Key="Light">
+			<!-- Non-opaque/transparent primary disabled color -->
+			<Color x:Key="M3MaterialPrimaryVariantLowThumbColor">#E9E5FA</Color>
+			<!-- Non-opaque/transparent white color that shows on white surfaces -->
+			<Color x:Key="M3MaterialSurfaceVariantLightColor">#F7F7F7</Color>
+		</ResourceDictionary>
+		<ResourceDictionary x:Key="Dark">
+			<!-- Non-opaque/transparent primary disabled color -->
+			<Color x:Key="M3MaterialPrimaryVariantLowThumbColor">#57507C</Color>
+			<!-- Non-opaque/transparent white color that shows on white surfaces -->
+			<Color x:Key="M3MaterialSurfaceVariantLightColor">#D7D7D7</Color>
+		</ResourceDictionary>
+	</ResourceDictionary.ThemeDictionaries>
+	
+
+
+	<!--  Toggle On  -->
+	<StaticResource x:Key="M3MaterialToggleSwitchOnButtonBrush"
+	                ResourceKey="PrimaryBrush" />
+	<StaticResource x:Key="M3MaterialToggleSwitchOnBackgroundBrush"
+	                ResourceKey="PrimaryVariantLightBrush" />
+
+	<!--  Toggle Off  -->
+	<StaticResource x:Key="M3MaterialToggleSwitchOffButtonBrush"
+	                ResourceKey="OnPrimaryBrush" />
+	<StaticResource x:Key="M3MaterialToggleSwitchOffBackgroundBrush"
+	                ResourceKey="OnSurfaceLowBrush" />
+
+	<SolidColorBrush x:Key="AndroidSelectControlToggleOffButtonBrush"
```

Comment:
`AndroidSelectControlToggleOffButtonBrush` is not use I think, can be removed if it's the case ;)